docroot/themes/humsci/humsci_basic/src/scss/components/_card.scss

```diff
@@ -207,23 +207,25 @@
   }
 
   &__title {
+    margin-top: 0;
+    margin-bottom: hb-calculate-rems(14px);
+
+    @include grid-media-min('md') {
+      margin-bottom: hb-calculate-rems(16px);
+    }
+
+    @include grid-media-min('lg') {
+      margin-bottom: hb-calculate-rems(22px);
+    }
+
     h2,
     h3,
```

Comment:
now you can delete the other headings I think!

Reply:
(since it just uses h2 now?

Reply:
Oh yeah! 😆